<launch>
  <arg name="mav_name" default="firefly" />

  <node name="mav_saver" pkg="mav_saver" type="mav_saver_node" clear_params="true" output="screen" >
    <remap from="odometry" to="/$(arg mav_name)/ground_truth/odometry" />
    <param name="measurement_divisor" value="10" />
    <param name="obstacle_octomap_path" value="res/LeoC6.bt" />
    <param name="enable_collision_constraint" value="true" />
    <param name="kill_switch_port_name" value="/dev/ttyUSB_kill_switch" />
    <param name="serial_port" value="/dev/ttyUSB_$(arg mav_name)" />
  </node>

  <include file="$(find mav_simulator_demos)/launch/mav_interactive_marker_msf.launch" />
</launch>
